src/data/managers_db.py

```diff
@@ -1,5 +1,7 @@
-from src.data.data import DatabaseConnection
+'''this module is to store the calls to the manager collection'''
+import pymongo
```

Comment:
Why are you importing this?

Reply:
not sure why it reverted it will be taken care of with next commit